docs/processes/quality-assurance/testing-and-dev.md

```diff
@@ -0,0 +1,200 @@
+# QA Process - Testing and Development
+
+[Testing and Development](#testing-and-development
+
+- [Feature Development: where to start?](#feature-development-–-where-do-i-start)
+
+- [Test-Driven Development](#test-driven-development)
+  - [Writing Appropriate Tests](#determining-appropriate-tests)
+  - [Writing a Failing Test Case](#writing-a-failing-test-case)
+  - [Writing Code to Pass a Test](#writing-code-to-pass-your-test)
+  - [Refactoring Code](#refactoring-code)
+- [Epics and User Stories](#epics-and-user-stories)
+  - [Epics](#epics)
+  - [User Stories](#user-stories)
+- [Testing Templates](#testing-templates)
+
+Current Areas under construction:
+
+- [Refactoring Intro](#tbc-refactoring-intro)
+- [Waiting epic Template link](#tbc-template-link)
+- [Testing tools expansion](#tbc-testing-tools)
+
+# Testing and Development
+
+Testing should be built into the software development process.
+
+Thoth-Tech advocates for a [Test Driven Development](#test-driven-development) approach.
```

Comment:
**suggestion:** We could introduce the TDD acronym in the first use here. What do you think?

```suggestion
Thoth-Tech advocates for a [Test Driven Development](#test-driven-development) (TDD) approach.
```